cli/common/before.go

```diff
@@ -0,0 +1,63 @@
+package common
```

Comment:
Why is this file named `before` when it contains `Before` and `After`? Rename it to e.g. `common.go`?